gulpfile.babel.js

```diff
@@ -314,20 +335,62 @@ watch.description = '`gulp watch` watches for changes and runs tasks automatical
 
 /**
  * Main buildtask:
- * Run `gulp build` respectively `gulp build -prod`
+ * Run `gulp build` respectively `gulp build --production`
  */
 export const build = gulp.series(
 	clean,
 	gulp.parallel(processHtml, lint, images, clientScripts, vendorScripts, styles, bundleExternalCSS, copyStaticFiles, lintBootstrap, security, test)
 );
 build.description = '`gulp build` is the main build task';
 build.flags = {
-	'-prod': ' builds for production to `dist` directory.'
+	'--production': ' builds for production to `dist` directory.',
+	'-P': ' Alias for --production'
+};
+
+function bumpVersion() {
+	if (!hasBumpType()) {
+		onError(new Error(chalk.red('Please specify release type: gulp release --bump (major|minor|patch)')));
+	}
+
+	return gulp.src('./package.json')
+		.pipe(bump({type: args.bump}))
+		.on('error', onError)
+		.pipe(gulp.dest('./'))
+		.pipe(touch());
+}
+
+function createChangelog() {
+	return gulp.src('./CHANGELOG.md', {buffer: false})
+		.pipe(changelog({preset: 'angular'}))
+		.pipe(gulp.dest('./'));
+}
+
+function commitChanges() {
+	return gulp.src('.')
+		.pipe(git.add())
+		.pipe(git.commit(`Release ${semver.inc(pkgJson.version, args.bump)}`));
+}
+
+function createTag(done) {
+	const version = JSON.parse(fs.readFileSync('./package.json', 'utf8')).version;
```

Comment:
> Why not update pkgJson in the bump task instead of re-requiring the package.json
Feels weird.

Now bumping with:
```
semver.inc(pkgJson.version, args.bump);
```